docs/kb/rf-increase.rst

```diff
@@ -27,6 +30,8 @@ As a result, in order to make sure that you can keep on reading the old data wit
 
 After you run a repair, you can decrease the CL. If RF has only been changed in a particular Data Center (DC) only the nodes in that DC have to be repaired.
 
+For the drivers issue: after tablet migrations caused by RF change are finished client applications should be restarted.
```

Comment:
Missing comma: " are finished, client applications should be restarted."
Again, should say how the user can know when this should be done.